.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,4 @@
+<!-- Keep the progress checklist.  -->
 ## Progress
```

Comment:
We have a comment that reads, "If you are reporting a bug, please fill out the sections below. Otherwise, the sections below can be deleted." My concern with moving it to the bottom is people would delete it if they see that and are not reporting a bug.